### DISCUSSION TITLE
Soft Restart?

### DISCUSSION THREAD

**Discussion by shaunzyb** [AzuraCast/AzuraCast#5213] [General]
Hi

Does the soft restart work if there is a DJ - Streamer connected?

Every time I have restarted the output has been interrupted.

**Comment by Vaalyn** [comment]
Liquidsoap does not have a soft restart feature. When you use the reload-without-kicking-listeners feature in AzuraCast this works by letting Icecast push listeners to a fallback file that just contains static white noise while Liquidsoap is restarting and Icecast reloads it's configuration. Since DJs connect through LS they will be disconnected when using this feature.
